### DISCUSSION TITLE
zlib包无法使用

### DISCUSSION THREAD

**Discussion by AkiraYe567** [tw93/Pake#389] [Q&A] [answered]
纯小白，之前zlib可以正常使用今天发现不行了，打开界面如下，可以求一个更新吗
![Screen Shot 2023-04-16 at 16 37 43](https://user-images.githubusercontent.com/91537822/232280865-09de47b7-5d51-4eb6-9963-cfc76a2888f3.png)

**Comment by tw93** [comment, marked as answer]
防止喝茶，已经下掉

**Reply by AkiraYe567** [reply]
好的 谢谢！
